src/pages/pool/components/pool-compositions.tsx

```diff
@@ -1,50 +1,86 @@
-import { useEffect } from 'react';
-import tw from 'twin.macro';
+import tw, { styled } from 'twin.macro';
 
-import { Tab } from '~/components/tab';
-import { Table } from '~/components/tables';
+import { GraphSemiDonut } from '~/components/graph';
 
-import { useTableTotalComposition } from '~/pages/pool/hooks/components/table/use-table-total-composition';
+import { formatNumberWithComma } from '~/utils';
+import { IPool, ITokenComposition } from '~/types';
 
-import { useConnectedWallet } from '~/hooks/wallets';
-import { useTablePoolCompositionSelectTabStore } from '~/states/components/table/tab';
-import { IPool } from '~/types';
+import { useGraphTotalComposition } from '../hooks/components/graph/use-graph-total-composition';
 
+interface TokenInfoProps {
+  symbol: string;
+  balance: number;
+  value: number;
+}
+const TokenInfo = ({ token, index }: { token: TokenInfoProps; index: number }) => {
+  return (
+    <TokenInfoWrapper>
+      <TokenSymbolWrapper>
+        <TokenColorCircle index={index} />
+        <TokenSymbolText>Total {token.symbol}</TokenSymbolText>
+      </TokenSymbolWrapper>
+      <TokenValueWrapper>
+        <TokenBalance>
+          {formatNumberWithComma(Math.trunc(token.balance))}
+          {` `}
+          {token.symbol}
+        </TokenBalance>
+        <TokenValue>${formatNumberWithComma(Math.trunc(token.value))}</TokenValue>
+      </TokenValueWrapper>
+    </TokenInfoWrapper>
+  );
+};
 interface Props {
   pool: IPool;
 }
 export const PoolCompositions = ({ pool }: Props) => {
-  const { selectedTab, selectTab } = useTablePoolCompositionSelectTabStore();
-  const { evm, xrp } = useConnectedWallet();
-  const address = evm?.address || xrp?.address;
-
-  const tabs = address
-    ? [
-        { key: 'total-composition', name: 'Total composition' },
-        { key: 'my-composition', name: 'My pool share' },
-      ]
-    : [{ key: 'total-composition', name: 'Total composition' }];
-
-  const { data, columns } = useTableTotalComposition(pool.id);
+  const { poolData } = useGraphTotalComposition(pool.id);
 
-  useEffect(() => {
-    if (!address) selectTab('total-composition');
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [address]);
+  const graphData: ITokenComposition[] = poolData.map(data => ({
+    symbol: data.symbol,
+    weight: data.weight,
+    value: data.value,
+    balance: data.balance,
+  }));
```

Comment:
null 케이스 핸들링만 추가해주세여

```suggestion
  const graphData: ITokenComposition[] = poolData?.map(data => ({
    symbol: data?.symbol ?? '',
    weight: data?.weight ?? 0,
    value: data?.value ?? 0,
    balance: data?.balance ?? 0,
  })) ?? [];
```

Reply:
[null case & graph wrapper css](https://github.com/TeamHeimdallr/moai-web/pull/151/commits/2a4e7bd1fd11a069422878874034c682a6693c55)